### ISSUE TITLE
Bug: Open in new tab is showing for files in search results

### ISSUE THREAD

**Issue by lrstolk** [files-community/Files#14060]
### Description
(Please click the video links, I used multiple video)
Context menu shows wrong info on a file. At [0:28](https://github.com/files-community/Files/assets/35999180/d99f984f-2869-4954-9757-5353be3f4bee) 
![image](https://github.com/files-community/Files/assets/35999180/c22cdda0-0a8b-48eb-9077-563295f3bed7) 
If pressed "Open in new tab" it gives:
<img width="253" alt="image" src="https://github.com/files-community/Files/assets/35999180/34a4a6d2-016e-4de8-8ddd-0cbb64342c7a">. 
But this is the wrong context menu. 

In the debug.log the error at 18:06 ([0:31](https://github.com/files-community/Files/assets/35999180/d99f984f-2869-4954-9757-5353be3f4bee)) is not logged. As is the wrong context menu not logged. It happend earlier around 17:55, but not sure if that is same error. 

Again the wrong context menu plus wrong preview results happend: 
At [0:33](https://github.com/files-community/Files/assets/35999180/b0fe126e-2395-4c81-b853-fdae623c1151) is shows the wrong search results. At [0:38](https://github.com/files-community/Files/assets/35999180/b0fe126e-2395-4c81-b853-fdae623c1151) is does not show al the results (Windows file explorer shows more, including the "238"folder). At [0:04](https://github.com/files-community/Files/assets/35999180/b0fe126e-2395-4c81-b853-fdae623c1151) and [0:39](https://github.com/files-community/Files/assets/35999180/b0fe126e-2395-4c81-b853-fdae623c1151) it shows the wrong context menu. 

Also clicking on the parent folder botton a few times causes Files to crash. 
https://github.com/files-community/Files/assets/35999180/9f7ed021-1223-4a51-97d8-05d6b19c93c6

### Steps To Reproduce

Check this [video1](https://github.com/files-community/Files/assets/35999180/c22cdda0-0a8b-48eb-9077-563295f3bed7), [video2](https://github.com/files-community/Files/assets/35999180/b0fe126e-2395-4c81-b853-fdae623c1151), [video3](https://github.com/files-community/Files/assets/35999180/9f7ed021-1223-4a51-97d8-05d6b19c93c6). 

### Requirements

Fix the contaxt menu, so it shows the right context so the error could never occur. 

### Files Version

3.0.8.0

### Windows Version

10.0.22631.2715

### Log File

[debug.log](https://github.com/files-community/Files/files/13461988/debug.log)

**Comment by yaira2** [comment]
Thanks for the report

**Comment by yaira2** [comment]
@lrstolk what's the file extension for that file?

**Comment by yaira2** [comment]
@lrstolk

**Comment by lrstolk** [comment]
> @lrstolk what's the file extension for that file?

It is an .FLP file

**Comment by yaira2** [comment]
> Also I see you changed the title, but that is one of the many problems, are all solved?

It's a bit hard to follow what's going on as the steps to repro the issue weren't added. Aside from the "open in new tab" showing in search results, what's the other issue?

**Comment by lrstolk** [comment]
> > Also I see you changed the title, but that is one of the many problems, are all solved?
> 
> It's a bit hard to follow what's going on as the steps to repro the issue weren't added. Aside from the "open in new tab" showing in search results, what's the other issue?

That is due to it maybe being in the original post https://github.com/files-community/Files/issues/14058 
Problem: 
If you search for an item, the second time searching it shows the wrong results, as shown in the video, the second time I search for 238, it shows different/wrong results. 

I also found another weird thing. When I use the column layout and go through maps and search and press back, it forgets the previous map I was in. My directory is like map1/map2/map3, map 1 is the inital map in the column layout, I search and press back, it goes to map1 instead of map1/map2/map3.
